packages/jsii-pacmak/lib/targets/dotnet/filegenerator.ts

```diff
@@ -183,6 +184,6 @@ export class FileGenerator {
       // suffix is guaranteed to start with a leading `-`
       return `${assembly.version}${suffix}`;
     }
-    return assembly.version;
+    return toReleaseVersion(assembly.version, TargetName.DOTNET);
```

Comment:
Non-blocking: It's a consolation that in most languages (besides .NET), `toReleaseVersion()` is called once or from a single file. Is it simple to do the same with .NET?

Reply:
Maybe it would be but this could entail a significant refactor... And perhaps would reduce the overall quality of the .NET generator's code quality 🙃

Reply:
Alright-y. Not ideal but ok :)